Meta/Lagom/Fuzzers/FuzzTGALoader.cpp

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibGfx/TGALoader.h>
+#include <stddef.h>
+#include <stdint.h>
+
+extern "C" int LLVMFuzzerTestOneInput(uint8_t const* data, size_t size)
```

Comment:
If we want this fuzzer to run in oss-fuzz it needs to be added to Meta/Lagom/Fuzzers/CMakeLists.txt as
```
add_simple_fuzzer(FuzzTGALoader LibGfx)
```